MapboxCoreNavigation/RouteController.swift

```diff
@@ -998,26 +1020,29 @@ extension RouteController {
 
         return event.id.uuidString
     }
-    
+
     func enqueueFoundFasterRouteEvent() -> String {
         let timestamp = Date()
-        let eventDictionary = events.navigationRerouteEvent(routeController: self, eventType: FasterRouteFoundEvent)
-        
+        let eventDictionary = eventsManager.navigationRerouteEvent(routeController: self, eventType: FasterRouteFoundEvent)
+
         sessionState.lastRerouteDate = timestamp
-        
+
         let event = RerouteEvent(timestamp: Date(), eventDictionary: eventDictionary)
-        
+
         outstandingFeedbackEvents.append(event)
-        
+//        eventsManager.enqueueEvent(withName: MMEEventTypeNavigationReroute, attributes: event.eventDictionary)
+
         return event.id.uuidString
     }
 
     func checkAndSendOutstandingFeedbackEvents(forceAll: Bool) {
-        let now = Date()
-        let eventsToPush = forceAll ? outstandingFeedbackEvents : outstandingFeedbackEvents.filter {
-            now.timeIntervalSince($0.timestamp) > SecondsBeforeCollectionAfterFeedbackEvent
-        }
-        for event in eventsToPush {
+//        let now = Date()
+
+//        let eventsToPush = forceAll ? outstandingFeedbackEvents : outstandingFeedbackEvents.filter {
```

Comment:
This is the one thing that needs to be reconciled before this set of changes can merge -- do we really need to do this event delay filtering? If so, we need to adjust the testing strategy a bit to take it into account, otherwise this code needs to be cleaned up.